From f493d0ba1d2cd6c2fdd3c80fe20908c5a6a0001a Mon Sep 17 00:00:00 2001
From: Mike Gilbert <floppym@gentoo.org>
Date: Thu, 25 Nov 2021 12:28:07 -0500
Subject: [PATCH] airodump-ng: look for oui.txt in /usr/share/hwdata

This location is used by the hwdata package on various distros.

https://github.com/vcrhonek/hwdata

Signed-off-by: Mike Gilbert <floppym@gentoo.org>
---
 src/airodump-ng/airodump-ng.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/airodump-ng/airodump-ng.c b/src/airodump-ng/airodump-ng.c
index 63013450d..23171854e 100644
--- a/src/airodump-ng/airodump-ng.c
+++ b/src/airodump-ng/airodump-ng.c
@@ -102,6 +102,7 @@ static const char * OUI_PATHS[]
 	   "/usr/share/aircrack-ng/airodump-ng-oui.txt",
 	   "/var/lib/misc/oui.txt",
 	   "/usr/share/misc/oui.txt",
+	   "/usr/share/hwdata/oui.txt",
 	   "/var/lib/ieee-data/oui.txt",
 	   "/usr/share/ieee-data/oui.txt",
 	   "/etc/manuf/oui.txt",
